ext/RMagick/rmmain.c

```diff
@@ -254,24 +254,16 @@ Init_RMagick2(void)
     rm_ID_cur_image        = rb_intern("cur_image");
     rm_ID_dup              = rb_intern("dup");
     rm_ID_fill             = rb_intern("fill");
-    rm_ID_flag             = rb_intern("flag");
-    rm_ID_from_s           = rb_intern("from_s");
     rm_ID_Geometry         = rb_intern("Geometry");
-    rm_ID_GeometryValue    = rb_intern("GeometryValue");
-    rm_ID_has_key_q        = rb_intern("has_key?");
     rm_ID_height           = rb_intern("height");
     rm_ID_initialize_copy  = rb_intern("initialize_copy");
-    rm_ID_length           = rb_intern("length");
```

Comment:
@Watson1978 would this be a breaking change?

Reply:
no, just removed garbaged variable in C-lang land. 
If broken, it will fail the compiling.